prometheus_es_exporter/__init__.py

```diff
@@ -295,6 +295,10 @@ def main():
                         help='path to a CA certificate bundle. Can be absolute, or relative to the current working directory. If not specified, SSL certificate verification is disabled.')
     parser.add_argument('-p', '--port', type=int, default=9206,
                         help='port to serve the metrics endpoint on. (default: 9206)')
+    parser.add_argument('-u', '--user', default='',
+                        help='User for authentication. (default: no user)')
+    parser.add_argument('-P', '--password', default = '',
```

Comment:
Lets use `--basic-user` and `--basic-password`, with no short versions for now.

Please also drop the the `default` parameters - we want to be able to identify if they provided these arguments by checking for `None`.